docs/index.md

```diff
@@ -108,24 +108,25 @@ napari into a clean virtual environment using an environment manager like
 [venv](https://docs.python.org/3/library/venv.html).  For example, with `conda`:
 
 ```sh
-conda create -y -n napari-env python=3.8
+conda create -y -n napari-env python=3.9
 conda activate napari-env
 pip install "napari[all]"
 ```
 ````
 
-### From source
-
-To clone the repository locally and install in editable mode use
```

Comment:
since this was in the general "installing napari" document, and not the "contributing to napari" document, I'm changing the instructions for installing the current main branch to a one liner that goes directly to site-packages